### PR TITLE
user/k6: new package

### PR DESCRIPTION
**Pull request by dtrsh**

### CODE REVIEW

File: user/k6/template.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+pkgname = "k6"
+pkgver = "0.54.0"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Modern load testing tool, using Go and Javascript"
```
Comment (q66) [inline user/k6/template.py:6]: Load testing tool

File: user/k6/template.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+license = "AGPL-3.0-only"
+url = "https://github.com/grafana/k6"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "706089f6dd65b8e50cee71ac30d32b44be8ddb6c65acf8ff54b6a0729027148e"
+# k6 tests requires network connection
+options = ["!check"]
+
+
+def post_install(self):
+    self.install_license("LICENSE.md")
```
Comment (triallax) [inline user/k6/template.py:17]: agpl doesn't need install

Comment (dtrsh) [reply user/k6/template.py:17]: Seriously? I got an error during the local build

```
ERROR: license installation necessary but no license installed
```

Comment (q66) [reply user/k6/template.py:17]: agpl does need license installation, the lint is correct as is the install_license

**Pull request merged**